Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -965,7 +873,8 @@ static u32 TranslateBlockAddress(const u32 addr, const XCheckTLBFlag _Flag)
 }
 
 // Translate effective address using BAT or PAT.  Returns 0 if the address cannot be translated.
-u32 TranslateAddress(const u32 _Address, const XCheckTLBFlag _Flag)
+template <const XCheckTLBFlag _Flag>
```

Comment:
Oh, geez, you're right.

It uses all four values of the flag parameter; do you want me to explicitly instantiate all four? it feels a bit redundant but I'm fine with it if that's what you want